server/data/stock-market.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require("mongoose");
+const {stockMarketInfo} = require("../dal/stock-markets");
+
+const MARKET_DATA = [{
+    market_name: "Nasdaq",
+    stocks: [
+        {
+            ticker: "TSLA",
+            name: "Tesla"
+        },
+        {
+            ticker: "AMZN",
+            name: "Amazon"
+        },
+        {
+            ticker: 'GOOGL',
+            name: "Google"
+        },
+        {
+            ticker: 'FB',
+            name: "Facebook"
+        },
+        {
+            ticker: 'TWTR',
+            name: "Twitter"
+        },
+        {
+            ticker: 'UBER',
+            name: "Uber Technologies Inc"
+        }
+    ]
+}]
+
+for (const data of MARKET_DATA) {
+    const {market_name, stocks} = data
+
+    stockMarketInfo.find({market_name: market_name}).then((doc) => {
+        if (doc.length === 0) {
+            var stockMarketData = new stockMarketInfo(data)
```

Comment:
Shouldn't this be let, because it's a block scoped declaration rather than function scoped used for var